src/Umbraco.Web.BackOffice/ModelBinders/BlueprintItemBinder.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Umbraco.Core.Hosting;
+using Umbraco.Core.Mapping;
+using Umbraco.Core.Models;
+using Umbraco.Core.Serialization;
+using Umbraco.Core.Services;
+using Umbraco.Core.Services.Implement;
+using Umbraco.Web.Models.ContentEditing;
+
+namespace Umbraco.Web.BackOffice.ModelBinders
+{
+    internal class BlueprintItemBinder : ContentItemBinder
+    {
+        private readonly ContentService _contentService;
+
+        public BlueprintItemBinder(IJsonSerializer jsonSerializer, UmbracoMapper umbracoMapper, IContentService contentService, IContentTypeService contentTypeService, IHostingEnvironment hostingEnvironment, ContentService contentService2) : base(jsonSerializer, umbracoMapper, contentService, contentTypeService, hostingEnvironment)
+        {
+            _contentService = contentService2;
```

Comment:
This maybe a historical thing but why do we need 2 content services?

Reply:
@bergmania yes good point, i assume it was due to internals but these should be fixed in netcore?

Reply:
Good point.. I have removed the concrete type